### PR TITLE
rgw: clean some space line and format some code of lc

### PR DESCRIPTION
**Pull request by XiangxiangMeng**
Signed-off-by: xiangxiang <xiangxiang@xsky.com>

### CODE REVIEW

File: src/rgw/rgw_lc.cc [hunk 1]
```diff
@@ -30,11 +30,9 @@ bool LCRule::valid()
 {
   if (id.length() > MAX_ID_LEN) {
     return false;
-  }
-  else if(expiration.empty() && noncur_expiration.empty() && mp_expiration.empty() && !dm_expiration) {
+  } else if (expiration.empty() && noncur_expiration.empty() && mp_expiration.empty() && !dm_expiration) {
```
Comment (mattbenjamin) [inline src/rgw/rgw_lc.cc]: if we're going to do a spacing cleanup, it would be nice to break >80 column expressions

**Comment by mattbenjamin** [comment]
let's give this one batch run

**Comment by ivancich** [comment]
jenkins test make check arm64

**Comment by ivancich** [comment]
@XiangxiangMeng Could we get a rebase?

**Pull request closed without merging**